build/Dockerfile.operator

```diff
@@ -24,7 +24,7 @@ RUN CGO_ENABLED=1 GOOS=linux go build \
 	github.com/IBM/ibm-block-csi-operator/cmd/manager
 
 
-FROM registry.access.redhat.com/ubi7/ubi-minimal:7.8-237
+FROM registry.access.redhat.com/ubi7/ubi-minimal:7.9-224
```

Comment:
why not `latest`?

Reply:
`latest` is not deterministic.
if you find any `latest` in our projects, please let me know

Reply:
what do you mean by "not deterministic"?

Reply:
with `latest`, building the same commit at different times could yield different outputs - which is bad for CI stability